CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 
 - Added the `useMutation.throwOnError` and corresponding `queryConfig.throwOnError` option to configure whether the `mutate` function rethrows errors encountered in the mutation function
 - Added the `useMutation.useErrorBoundary` and corresponding `queryConfig.useErrorBoundary` option to configure whether mutation errors should be thrown during the render function and propagated to the nearest error boundary. This option will default to the same value as `queryConfig.suspense` if not defined otherwise
+- Added a new `reset` function for `useMutation` which will revert the hook's state back to the initial `null` state
```

Comment:
While I believe this got merged as `v0.3.26` in https://github.com/tannerlinsley/react-query/commit/3a8bda84423ef8dd6ea1e37b9cc1b418698a7329, this would be more of a feature in semver terms. Let me know if you want me to move this to the `v0.3.26` section